geps/overview.md

```diff
@@ -11,15 +11,36 @@ process for the main Kubernetes project:
 
 ## Process
 
-### 1. Circulate the Idea
+This diagram shows the state diagram of the GEP process at a high level, but the details are below.
+
+<div align="center">
+```mermaid
+flowchart TD
+    D([Discuss with<br />the community]) --> C
+    C([Issue Created]) --> Provisional
+    Provisional -->|GEP Doc PR<br />done| Implementable
+    Provisional -->|If practical <br /> work needed| Prototyping
+    Prototyping -->|GEP Doc PR<br />done| Implementable
+    Implementable -->|Gateway API<br />work completed| Experimental
+    Experimental -->|Implemented in<br />controllers| Standard
```

Comment:
Since the Gateway API does not ship controllers, what does "Implemented in controllers" mean? Maybe we can say something like "implemented in the API"?

Reply:
The idea here is that things aren't moved to standard until enough implementations have finished working on them. "Implemented in implementations" didn't sound good, but maybe "Supported in multiple implementations" is a better way of phrasing that will make it clearer and more standard.

Reply:
I updated, and added a note about conformance tests.